frameworks/hdfs/src/main/dist/fix-keytab-bug.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -x
+export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
+$JAVA_HOME/bin/java -jar krbtest.jar $1
```

Comment:
Can you rename the java jar to something more appropriate? `krbtest` doesn't sound right.

Also everywhere you're calling this script already includes 
```
set -x 
export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
```
can we move these two other commands directly into `svc.yml` and get rid of this script entirely?